deployments/common/image/common-scripts/unset-notebook-kernel

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env python
```

Comment:
Part of the process for pushing notebooks to "webinar-specific" branches is to unset the kernel spec (idea being that if they download a notebook directly from current_branch or main, they're unlikely to have a kernel named "jwebbinar" for example).

It was requested :)

Reply:
That makes perfect sense.   I was worried about setting kernels in git because those decisions and names are really unique to us and the notebooks are not.   But if unsetting works,  great!